JetNews/app/src/main/java/com/example/jetnews/ui/interests/SelectTopicButton.kt

```diff
@@ -50,24 +51,30 @@ fun SelectTopicButton(
 
 @Composable
 private fun DrawSelectTopicButtonOn() {
-    DrawShape(
-        shape = CircleShape,
-        color = MaterialTheme.colors().primary
-    )
-    DrawVector(vectorResource(R.drawable.ic_check))
+    Box(
+        modifier = DrawBackground(
```

Comment:
Box has a shape and backgroundColor params, consider using it

Reply:
+1 thanks!